www/guanlecoja-ui/package.json

```diff
@@ -10,7 +10,7 @@
         "build": "rimraf dist && webpack --bail --progress --profile --env dev && webpack --bail --progress --profile --env prod",
         "test": "karma start",
         "test-watch": "karma start --auto-watch --no-single-run",
-        "yarn-update-local": "yarn remove buildbot-build-common && yarn add --dev ../build_common"
+        "yarn-update-local": "yarn remove buildbot-build-common && yarn add --dev ../build_common && git checkout yarn.lock && git checkout yarn.lock"
```

Comment:
`yarn link` registers a global reference to the package, so it means the developer wouldn't be able to work with multiple projects using different versions of the linked package.

I didn't want to checkout package.json because it's expected to be the same after this operation. If it's not, then it's best left to the developer to investigate.